lms/static/scripts/frontend_apps/components/SubmitGradeForm.tsx

```diff
@@ -237,9 +213,12 @@ export default function SubmitGradeForm({
               id={gradeId}
               elementRef={inputRef}
               onInput={e => handleInput(e, 'grade')}
-              type="text"
+              type={'number' as any}
```

Comment:
Current frontend-shared release does not support `Input[type="number"]`. Next one will https://github.com/hypothesis/frontend-shared/pull/1248